xtext/Dockerfile

```diff
@@ -95,4 +95,7 @@ RUN chmod +x /editorserver/start.sh
 # deploy tool static files
 COPY --from=toolstaticbuild /usr/src/mdenet-tool/dist/ROOT.war /usr/local/tomcat/webapps/ROOT.war
 
+# Cron time for scheduled stop
+ENV XTEXT_ES_STOP_CRON_TIME="* 4 * * *"
```

Comment:
Doesn't this contradict the documented default?

Reply:
Why did you specifically choose stopping at 4am?

Reply:
My bigger problem is that the documentation says the default is not to stop the container, but the dockerfile then sets a time for stopping anyway...

Reply:
I see the confusion,  defaults column removed.